src/freenas/usr/lib/netdata/python.d/disktemp.chart.py

```diff
@@ -0,0 +1,36 @@
+from bases.FrameworkServices.SimpleService import SimpleService
+from copy import deepcopy
+
+from middlewared.utils.disks import get_disks_for_temperature_reading, get_disks_temperatures
```

Comment:
Why not use middleware client for that like we did with collectd plugins?

Reply:
@themylogin netdata user cannot access the middleware client sadly..

Reply:
@Qubad786 why?